services/director/src/simcore_service_director/config.py

```diff
@@ -22,6 +22,21 @@
 API_VERSION: str = "v0"
 API_ROOT: str = "api"
 
+

```

Comment:
we should plan moving and dropping the director ASAP. Let's create a PR with the steps needed and we can follow up on that. What do you think?